mvp/README.md

```diff
@@ -0,0 +1,19 @@
+
+**User-Centric** design enables the continuous delivery of value to users. This means that putting a good, functional, tool in the hands of users quickly is more valuable than delivering a perfect tool when it's too late.
```

Comment:
Yes, its probably not a good idea to keep thinks late. Because we don't know when will the perfect tool be made , and waiting for such a tool is a waste of time and time is precious.